src/Raven.Studio/typescript/viewmodels/manage/adminLogs.ts

```diff
@@ -227,14 +228,19 @@ class adminLogs extends viewModelBase {
         return item.includes("EXCEPTION:") || item.includes("Exception:") || item.includes("FATAL ERROR:");
     }
     
+    private isCustomizedItem(item: string): boolean {
+        return (item.includes("Connection established") || item.includes("Connection closed") || item.includes("Connection paused"));
+    }
+    
     // noinspection JSMethodCanBeStatic
     itemHtmlProvider(item: string) {
-        const errorClass = this.hasError(item) ? "class='bg-danger'" : "";
-
-        return $("<pre class='item'></pre>")
+        const errorClass = this.hasError(item) ? `class="bg-danger"` : "";
```

Comment:
maybe create helper method to get decorator class. It will return "bg-danager", "custom-msg" or "" ?

Reply:
done